include/fast_float/ascii_number.h

```diff
@@ -6,11 +6,13 @@
 #include <cstdint>
 #include <cstring>
 #include <iterator>
-#include <version>
 
+#if defined(__has_include) && __has_include(<version>)
```

Comment:
This is not portable code, please see the rationale at https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005finclude.html

Reply:
fixed